local-cli/runAndroid/runAndroid.js

```diff
@@ -98,13 +93,32 @@ function buildAndRun(args) {
         args.flavor[0].toUpperCase() + args.flavor.slice(1)
       );
     } else {
-      gradleArgs.push('installDebug');
+      gradleArgs.push('install');
```

Comment:
Sure, the `build type` (`Release` or `Debug`) is now added in the line 99. By default it will append to the `install` + `Debug` unless the user pass as argument `--configuration release`. In that case it will append to the `install` + `Release`.

In the previous code version, it will always run the debug version.